doc/modules/preprocessing.rst

```diff
@@ -146,6 +148,61 @@ full formula is::
 
     X_scaled = X_std / (max - min) + min
 
+:class:`MaxAbsScaler` works in a very similar fashion, but scales data so
+it lies within the range ``[-1, 1]``, and is meant for data
```

Comment:
Yet this is not true of the following example. Either qualify the statement or add a trim option to the scaler.